asab/api/service.py

```diff
@@ -60,15 +59,10 @@ def _build_zookeeper_adv_data(self):
 		return adv_data
 
 
-	def _initialize_web(self, listen):
+	def _initialize_web(self):
 		websvc = self.App.get_service("asab.WebService")
 
-		# Create a dedicated web container
-		container = asab.web.WebContainer(
-			websvc, "asab:web",
-			config={"listen": listen}
-		)
-		# TODO: refactor to use custom config section, instead of explicitly passing "listen" param?
+		container = websvc.Container
```

Comment:
@byewokko I don't see how this is propagated. When is `websvc.Container` created?

Reply:
yes, this is definitely a mistake. I didn't understand Aleš's comment above.